Caltech-IPAC/firefly
# DM-14505: fixed mult table coverage not working correctly and a caching issues

- All tables load correctly for coverage
- URL based request now cache correctly
- removed FileHolder.java, a interface that was only implement by on class
- removed useHiPSForCoverage option, now the is always on.
- fixed a problem an exception when loading wise images from metaConvert
- remove CoverageChooser since it is not longer used.

_to test cache bug:_

- load a image using the URL option, such as http://web.ipac.caltech.edu/staff/roby/demo/wise-m51-band2.fits
- then load the same image again.  It should come from the cache.

_to test coverage bug:_
- use wise to load two different results
- They should both be overlayed.
